### PR TITLE
[All] (Centralized deployment) Note Mac supported

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Note Mac support based on [Centralized Deployment article](https://docs.microsoft.com/en-us/office365/admin/manage/centralized-deployment-of-add-ins#client-requirements)

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1717322](https://github.com/OfficeDev/office-js-docs-pr/pull/899/commits/1717322eb8ea4e0b5f75ced67f791e70effb8a8b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904161613139708-899%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=oFKviFkf4QgOpPKrWiaJFMQX4UvyjWyssPZqkZuycEs%3D&st=2019-04-16T16%3A09%3A30Z&se=2019-05-17T16%3A14%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by ElizabethSamuel-MSFT** [comment]
Addresses issue https://github.com/OfficeDev/office-js-docs-pr/issues/895

File: docs/publish/centralized-deployment.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ The Office 365 admin center makes it easy for an administrator to deploy Office
...
 
-Centralized Deployment is the recommended way for an Office 365 admin to deploy Office Add-ins within an organization, provided that the organization meets all requirements for using Centralized Deployment. For information about how to determine if your organization can use Centralized Deployment, see [Determine if Centralized Deployment of add-ins works for your Office 365 organization](https://support.office.com/article/Determine-if-Centralized-Deployment-of-add-ins-works-for-your-Office-365-organization-B4527D49-4073-4B43-8274-31B7A3166F92).
+Centralized Deployment is the recommended way for an Office 365 admin to deploy Office Add-ins within an organization, provided that the organization meets all requirements for using Centralized Deployment. For information about how to determine if your organization can use Centralized Deployment, see [Determine if Centralized Deployment of add-ins works for your Office 365 organization](/office365/admin/manage/centralized-deployment-of-add-ins).
```
Comment (AlexJerabek) [inline docs/publish/centralized-deployment.md]: This link doesn't work on the staging site. I think it should still be the link to the support article.

File: docs/publish/centralized-deployment.md [hunk 2]
```diff
@@ -142,4 +142,4 @@ However, for Outlook 2016 or later, users can do the following:
 
 ## See also
 
-- [Determine if Centralized Deployment of add-ins works for your Office 365 organization](https://support.office.com/article/Determine-if-Centralized-Deployment-of-add-ins-works-for-your-Office-365-organization-b4527d49-4073-4b43-8274-31b7a3166f92)
+- [Determine if Centralized Deployment of add-ins works for your Office 365 organization](/office365/admin/manage/centralized-deployment-of-add-ins)
```
Comment (AlexJerabek) [inline docs/publish/centralized-deployment.md]: Same link comment.

File: docs/publish/centralized-deployment.md (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply docs/publish/centralized-deployment.md]: @AlexJerabek: If you try the support link, it redirects to this (https://docs.microsoft.com/en-us/office365/admin/manage/centralized-deployment-of-add-ins?redirectSourcePath=%252farticle%252fDetermine-if-Centralized-Deployment-of-add-ins-works-for-your-Office-365-organization-B4527D49-4073-4B43-8274-31B7A3166F92&view=o365-worldwide) so I cut out the middle man. From the staging site, you may need to switch to master branch or go to PROD site.

Comment (AlexJerabek) [reply docs/publish/centralized-deployment.md]: Okay. For some reason I didn't think we could relative links outside of Office Add-ins content.

**Pull request merged**